Intents Extension/IntentHandler.swift

```diff
@@ -25,6 +25,8 @@ class IntentHandler: INExtension {
             return ListMilestonesIntentHandler()
         case is ConfigurationIntent:
             return ConfigurationIntentHandler()
+        case is CampaignLockScreenConfigurationIntent:
+            return CampaignConfigurationIntentHandler()
```

Comment:
But not the Fundraising one?

Reply:
That doesn't have anything that needs custom resolution - this is only necessary here to return the list of Campaigns from the database for the "Campaign?" option